cluster-provision/gocli/cmd/run.go

```diff
@@ -71,6 +71,7 @@ func NewRunCommand() *cobra.Command {
 	run.Flags().String("log-to-dir", "", "enables aggregated cluster logging to the folder")
 	run.Flags().Bool("enable-ceph", false, "enables dynamic storage provisioning using Ceph")
 	run.Flags().String("docker-proxy", "", "sets network proxy for docker daemon")
+	run.Flags().Bool("serial-pty", false, "attach serial to /dev/pts/ device")
```

Comment:
I guess we could just always add it.

Reply:
What do you think?

Reply:
I think so, in case we see problems we can go back to optional.